src/popup/pages/Index.vue

```diff
@@ -141,14 +142,13 @@ export default defineComponent({
 
     async function createWallet() {
       isWalletNew = true;
-      setGeneratedMnemonic();
+      await setMnemonicAndInitializeAuthentication(generateMnemonic());
       addRawAccount({
         isRestored: false,
         protocol: PROTOCOLS.aeternity,
         type: ACCOUNT_TYPES.hdWallet,
       });
       router.push(loginTargetLocation.value);
-      openEnableSecureLoginModal();
```

Comment:
It would no longer suggest user to set up a secure login. Are you sure about this changes?